cmd/algocfg/profileCommand.go

```diff
@@ -91,13 +91,69 @@ var (
 		},
 	}
 
+	hybridRelay = configUpdater{
+		description: "Relay consensus messages across both ws and p2p networks, also support recent catchup.",
+		updateFunc: func(cfg config.Local) config.Local {
+			// WS relay config defaults
+			cfg.MaxBlockHistoryLookback = 22000 // Enough to support 2 catchpoints with some wiggle room for nodes to catch up from the older one
+			cfg.CatchpointFileHistoryLength = 3
+			cfg.CatchpointTracking = 2
+			cfg.EnableLedgerService = true
+			cfg.EnableBlockService = true
+			cfg.NetAddress = ":4160"
+			// This should be set to the public address of the node if public access is desired
+			cfg.PublicAddress = "PLEASE_SET_ME"
```

Comment:
here and below
```suggestion
```
this should not be set otherwise config checker cannot detect missing PublicAddress for hybrid mode

Reply:
Going to address this in the identity deduping PR